### PR TITLE
Add repro for gh10741

### PR DESCRIPTION
**Pull request by gridbugs**
Repro for https://github.com/ocaml/dune/issues/10741

### CODE REVIEW

File: test/blackbox-tests/test-cases/pkg/oneshot-webserver-large-file-curl-error.t [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Reproduction for an issue with the oneshot webserver used for testing
+package management, where attempting to send a large file causes the
+connection to be reset before the transfer completes.
+
+Make a large file:
+  $ dd if=/dev/zero of=./large-file iflag=fullblock,count_bytes status=none count=20M
```
Comment (moyodiallo) [inline test/blackbox-tests/test-cases/pkg/oneshot-webserver-large-file-curl-error.t:6]: I have tested on my machine, and it starts failing from 14M, yeah 20M could be better number.

**Review by moyodiallo — APPROVED** [review]

**Review by rgrinberg — CHANGES_REQUESTED** [review]
Can you disable this test by default? Writing an additional 20 megs every time we run the test seems a bit annoying.

File: test/blackbox-tests/test-cases/pkg/dune [hunk 2]
```diff
@@ -44,7 +44,8 @@
   e2e
   source-caching
   tarball
-  extra-sources))
+  extra-sources
+  oneshot-webserver-large-file-curl-error))
```
Comment (rgrinberg) [inline test/blackbox-tests/test-cases/pkg/dune:48]: This test depends on curl as well

**Comment by rgrinberg** [comment]
Actually, the file size didn't need to be 20M to repro the problem. If it's 1M, then it should be enough to leave on by default.

**Comment by gridbugs** [comment]
Ok I'll change it to 1M. On my machine I can't reproduce the problem with a 1M file, but github actions seem to be more sensitive to this issue than my machine is.

**Comment by gridbugs** [comment]
This is no longer needed as the issue was fixed in https://github.com/ocaml/dune/pull/10753

**Pull request closed without merging**